citi-onboarding/Delta-odonto
# Feature/slide our blog

## What i did
The section Our blog

# How to test

- In the folder Delta-odonto run the command, check if you are in the branche develop, if you are not switch to develop with the command

`git checkout devlop`

- In the branch develop run the command

`git pull`

- Commit the changes

- Now, go to branch feature/slide-OurBlog with the command

`git checkout feature/slide-OurBlog`

- Now you need to update this branch with the command

`git push origin develop`

- So you can see the section Our Blog are in your right position

- Test the carousel, clicking in the arrows and dots

- Change the width dimensions of the page to check if the section is responsive

- Click on the cards below 'Acesse conteúdos gratuitos:', you should see a Modal with a form
